features/post-with-keys.js

```diff
@@ -0,0 +1,18 @@
+var page = function () {
+    if (document.URL.startsWith("https://scratch.mit.edu/discuss")) {
+        return "f"; // F = forums
+    } else {
+        return "e"; // E = else (not forums)
+    }
+}
+
+
+document.addEventListener(keypress, (e) => {
```

Comment:
"keypress" must be a string, it isn't defined as anything else.

Reply:
Oh shoot. How did I not notice the bug? 😅